helm-charts/auth-apisix/README.md

```diff
@@ -0,0 +1,56 @@
+# Authentication and Authorization with APISIX and OIDC based Identity provider (Keycloak)
+
+Follow the steps to enable authentication and authorization of OPEA services using APISIX api gateway and Identity provider like keycloak
+
+## Prerequisites
+
+1. Make sure ChatQnA service is running and accessible locally
+
+2. Run keycloak, setup a realm with OIDC based authentication and add users with passwords for each user.
+
+Steps to start keycloak from official keycloak helm chart
+
+```sh
+# Prerequisite: Create a PersistentVolume of 9Gi for keycloak-postgress with RWO access (to persist updated keycloak configuration)
```

Comment:
Do we need to add a manifest to show how to create such persistentvolume or refer to some link? It's better that the readme is self-contained.